src/core.js

```diff
@@ -348,6 +348,16 @@ extend( QUnit, {
 				return type.toLowerCase();
 		}
 		if ( typeof obj === "object" ) {
+			if ( obj instanceof Int8Array ||
+				obj instanceof Uint8Array ||
```

Comment:
Multi-line if-statements get extra indent, so all of these `instanceof` lines except the first need one more tab.